### ISSUE TITLE
Javadoc issue in 2.1.9

### ISSUE THREAD

**Issue by trask** [immutables/immutables#251]
I get this javadoc error using 2.1.9:

```
D:\git\trask\glowroot\common\target\generated-sources\annotations\org\glowroot\common\live\ImmutableTracePoint.java:322: error: reference not found
[ERROR] * Fill a builder with attribute values from the provided {@link TracePoint} instance.
```

With 2.1.8, the javadoc correctly references the nested type:

```
    /**
     * Fill a builder with attribute values from the provided {@link LiveTraceRepository.TracePoint} instance.
     * ...
     */
    public final Builder copyFrom(LiveTraceRepository.TracePoint instance) {
        ...
    }
```

With 2.1.9, the javadoc leaves off the outer type:

```
    /**
     * Fill a builder with attribute values from the provided {@link TracePoint} instance.
     * ...
     */
    public final Builder copyFrom(LiveTraceRepository.TracePoint instance) {
        ...
    }
```

It appears only the javadoc for `copyFrom()` has this problem.

Let me know if you have any trouble reproducing and I'll submit a test case.  Thanks.

**Comment by elucash** [comment]
There was some changes recently, fixing issues, and I got most places to use `{@code ` rather that `{@link ` there.  The takeaway is that I should enable javadoc building (failing on errors) during regular build at least for fixture module. Thank you for spotting this! The fix will be released shortly.

**Comment by elucash** [comment]
Released in 2.1.10. Thanks!

**Comment by trask** [comment]
Thanks for the quick fix/release as always.
